src/source/ObjcGenerator.scala

```diff
@@ -417,6 +420,44 @@ class ObjcGenerator(spec: Spec) extends Generator(spec) {
       }
       w.wl
 
+      w.wl("- (NSDictionary *)toDict")
+      w.braced {
+        w.wl("#define _djinni_hide_null_(_o_) ((_o_)?(_o_):([NSNull null]))")
```

Comment:
i would love to add it to a common file, but unfortunately the generated ObjC code, doesn't define a file that other files include (yet)